js/private/js_helpers.bzl

```diff
@@ -138,7 +134,7 @@ def gather_runfiles(
     Args:
         ctx: the rule context
 
-        sources: list or depset of files which should be included in runfiles
+        sources: depset which should be included in runfiles
```

Comment:
Note that this one switched to `depset` instead of `list` like the others. It seems like all use cases of `gather_runfiles` in rules_js+ts are already passing a `depset` (except the one in rules_js that was just passing an empty list...).